docs/compass/02-01-components.md

```diff
@@ -0,0 +1,45 @@
+---
+title: Components
+type: Architecture
+---
+
+![Components](./assets/components.svg)
```

Comment:
![Screen Shot 2019-10-04 at 13 42 30](https://user-images.githubusercontent.com/7155799/66205116-d98a6580-e6ac-11e9-947e-d8662f40876e.png)

Please fix the diagram